netctl/http.go

```diff
@@ -54,7 +54,10 @@ func postMap(ctx *cli.Context, url string, jsonMap map[string]interface{}) {
 
 	resp, err := client.Post(url, "application/json", bytes.NewBuffer(content))
 	if err != nil {
-		writeBody(resp, ctx)
+		if resp != nil {
```

Comment:
I am just wondering if there is a real case to even print a `resp` if `err` was not nil? I think most libraries follow this semantic that if they return a non nil error than caller should not interpret the other return values. `err` being not nil here also most likely implies that POST itself failed and we should not get anything back from the server so we can skip trying to write a nil response back.